client/pages/Dashboard/index.jsx

```diff
@@ -116,7 +128,14 @@ class Dashboard extends Component {
    */
   render() {
     const { location: { search } } = this.props;
-    const { activeUser } = this.state;
+    const { activeUser, categoryList } = this.state;
```

Comment:
Just curious, Why is `categoryList state` not initialized in the constructor ?

Reply:
Done